src/api-binder/poll.ts

```diff
@@ -188,7 +186,11 @@ const poll = async (
 		await update();
 		// Reset fetchErrors because we successfuly updated
 		fetchErrors = 0;
-	} catch {
+		lastFetch = process.hrtime();
```

Comment:
It's probably worth renaming to
```suggestion
		lastSuccessfulFetch = process.hrtime();
```
to make the semantics clear